tests/conftest.py

```diff
@@ -84,3 +69,10 @@ def slices2():
         slices.append(_slice)
 
     return slices
+
+
+@pytest.fixture(scope="function", autouse=True)
```

Comment:
I think this is useful in all cases, and avoids `np.seed()` calls in individual tests.